### PR TITLE
Adding trove classifiers.

### PR DESCRIPTION
**Pull request by nickjacobson**
Adding trove classifiers for Python 2 only.

### CODE REVIEW

**Comment by shortdudey123** [comment]
Replaces https://github.com/python-diamond/Diamond/pull/490

File: setup.py [hunk 1]
```diff
@@ -144,5 +144,9 @@ def pkgPath(root, path, rpath="/"):
     scripts=['bin/diamond', 'bin/diamond-setup'],
     data_files=data_files,
     install_requires=install_requires,
+    classifiers=[
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        ],
```
Comment (shortdudey123) [inline setup.py]: Indent this to the same level as `classifiers` for consistency within the file

Comment (nickjacobson) [inline setup.py]: I tried it and that broke the regression tests, so I reverted the indentation level back.

Comment (shortdudey123) [inline setup.py]: hmm interesting
it doesn't complain [here](https://github.com/nickjacobson/Diamond/blob/c1e74905303bc3f4d6d2ce191212b2c923e14694/setup.py#L36) or [here](https://github.com/nickjacobson/Diamond/blob/c1e74905303bc3f4d6d2ce191212b2c923e14694/setup.py#L43)

Comment (nickjacobson) [inline setup.py]: ok, you meant to just dedent the line with the bracket, doing so.

**Comment by josegonzalez** [comment]
Thanks for the pull request!

**Pull request merged**